### ISSUE TITLE
[AVT], StackedBarChart/ DonutChart/ LineChart, Elements with WAI-ARIA grouping roles must provide an 'aria-labelledby' or 'aria-label'

### ISSUE THREAD

**Issue by PAnilReddy** [carbon-design-system/carbon-charts#636]
Charts version
"@carbon/charts": "^0.30.10",

Elements with WAI-ARIA grouping roles must provide an 'aria-labelledby' or 'aria-label' violations are seen in the 2 charts we use StackedBarChart/ DonutChart/ LineChart
3 violations in LineChart
3 violations in DonutChart
6 violations in StackedBarChart

![image](https://user-images.githubusercontent.com/44162180/82662317-45750a80-9c4b-11ea-9244-0a4b1e38e697.png)

![image](https://user-images.githubusercontent.com/44162180/82662474-88cf7900-9c4b-11ea-8ba3-b0121fee6952.png)
